### PR TITLE
Run on codesandbox

### PR DESCRIPTION
**Pull request by KianiDev**
Added support for CodeSandbox

### CODE REVIEW

File: .codesandbox/tasks.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
+    "setupTasks": [
+      {
+        "name": "Install Dependencies",
+        "command": "npm install"
+      }
+    ],
+    // Adding Start task
+    "tasks": {
+      "npm install; npm start": {
```
Comment (SheepTester) [inline .codesandbox/tasks.json:10]: I don't know how Codesandbox tasks work, but if there's already an `npm install` task above, then does it also need to `npm install` here too?

File: .codesandbox/tasks.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+        "name": "Start",
+        "command": "npm install; npm start",
+        "preview": {
+          "port": 3000
+        }
+      }
+    }
+  }
```
Comment (SheepTester) [inline .codesandbox/tasks.json]: The file should end in a new line to be consistent with the rest of the repo

**Review by SheepTester — COMMENTED** [review]
Could you add a button to open the repo in Codespaces to the README, if such a thing exists? Since there's already a "Run in Replit" button, I think it'd fit right next to it

File: .codesandbox/tasks.json (hunk 1 above)
Comment (KianiDev) [reply .codesandbox/tasks.json:10]: I got an error without that. But I have to test again and see if the startup npm install is working or not.

Comment (KianiDev) [reply .codesandbox/tasks.json:10]: Do you want to change that to `npm update` instead?

Comment (KianiDev) [reply .codesandbox/tasks.json:10]: Changed the startup task to start the server automatically and removed `npm install` from the start task.

File: .codesandbox/tasks.json (hunk 2 above)
Comment (KianiDev) [reply .codesandbox/tasks.json]: Sorry, I fixed that.

**Comment by KianiDev** [comment]
> Could you add a button to open the repo in Codespaces to the README, if such a thing exists? Since there's already a "Run in Replit" button, I think it'd fit right next to it

https://codesandbox.io/p/github/SheepTester/primitive-cloud-server
This link can do it.

**Review by SheepTester — APPROVED** [review]
looks good, the link works for me so I added [a badge to the readme](https://github.com/codesandbox/codesandbox-client/issues/89)

**Pull request merged**